testing/recipes/publish_mlperf_ssd_image.sh

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+local_name="ssd"
+tag="nvidiadl-ssd-training-benchmark"
+
+repo="https://github.com/openshift-psap/training_results_v0.7.git"
+path="NVIDIA/benchmarks/ssd/implementations/pytorch"
```

Comment:
can you add a comment saying that we use a custom commit to avoid an image build bug?

I'll repush my commit https://github.com/kpouget/training_results_v0.7/commit/168f1fae6d91082d775ab698b1cb49cc3bc4d27a in this branch with the right git message